### ISSUE TITLE
about PSNR_b

### ISSUE THREAD

**Issue by SpilutX** [ByChelsea/CMOS#3]
I want to know that during the third stage of training, with torch.no_grad has been added to the CMOS network, and the PSNR_b result still changes every time it is tested. Why is this?

**Comment by ByChelsea** [comment]
I need some time to check on this. I will come back to answer the question once I find the reason.

---------------------------

Hi, I've found the reason.

The CMOS network contains BN layers, and the means and variances of the BN layers are different from other parameters that require gradient updates. We must use `.eval()` to freeze them; otherwise, they will still change during the training process.

Thank you for pointing out the issues. I will correct these bugs and provide the updated pre-trained model. By the way, to be honest, the model structure in this paper is rather cumbersome. Sometimes, the model structure doesn't actually need to be that complicated. But I think the idea of simulating space-variant blur, as well as the introduction of semantic information, is still worth considering.

I hope this helps, and feel free to ask if you have any other questions. :)
